content/_data/changelogs/weekly.yml

```
@@ -7719,6 +7719,134 @@
 
   # pull: 4732 (PR title: Fix hyperlink in maintainers doc)
 
+- version: "2.239"
+  date: 2020-06-01
+  banner: >
+    Jenkins 2.239 was not packaged or delivered.
+    All changes planned for 2.239 are included in 2.240.
+  changes:
+  - type: bug
+    category: internal
+    authors:
+    - kohsuke
+    message: |-
+      Jenkins 2.239 was not placed in the artifact repository or on the download site.
+
+- version: "2.240"
+  date: 2020-06-01
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4752
+    issue: 60924
+    authors:
+    - 95jonpet
+    - timja
+    - fqueiruga
+    references:
+      - issue: 60924
+      - pull: 4752
+      - url: https://github.com/jenkinsci/dark-theme
+        title: Dark Theme repository
+    message: |-
+      Release 'alpha' dark theme.
+  - type: major rfe
+    category: developer
+    pull: 4752
+    issue: 60924
+    authors:
+    - 95jonpet
+    - timja
+    - fqueiruga
+    references:
+      - issue: 60924
+      - pull: 4752
+      - url: https://github.com/jenkinsci/dark-theme
+        title: Dark Theme repository
+    message: |-
+      Developer: Add incubating support for CSS variables, useful in themes.
+  - type: rfe
+    category: rfe
+    pull: 4739
+    authors:
+    - res0nance
+    message: |-
+      Make CLI ManagementLink visible to users with Overall/SystemRead permission.
+  - type: rfe
+    category: rfe
+    pull: 4740
+    issue: 62421
+    authors:
+    - romenrg
+    message: |-
+      Improve accessibility by adding aria-labels for username and password fields in login form.
+  - type: rfe
+    category: rfe
+    pull: 4756
+    issue: 62434
+    authors:
+    - res0nance
+    message: |-
+      Make expandableTextbox compatible with System Read.
+  - type: rfe
+    category: rfe
+    pull: 4466
+    issue: 20052
+    authors:
+    - res0nance
+    message: |-
+      Reduce number of calls to getItems() and iterations through the jobList in ListView.
+  - type: rfe
+    category: rfe
+    pull: 4750
+    issue: 62477
+    authors:
+    - amuniz
+    message: |-
+      Remove subversion from suggested plugins in the setup wizard.
```

```suggestion
      Remove Subversion Plugin from suggested plugins in the setup wizard.
```

Also add a plugin link?